### DISCUSSION TITLE
Hiding "Item ID" / StickySidebar in AdminUI

### DISCUSSION THREAD

**Discussion by Francoois** [keystonejs/keystone#8332] [Questions]
Hello, 
I have found no way to hide the Sidebar. I can add elements to it with `fieldPosition : sidebar`, but I would like to remove "Item ID" field that I'm not using and use the space it occupies. Is there a way to do this? 

Using document editor with the sidebar ruins the experience if you edit complex documents on a standard sized screen, even on a big monitor.
 
For now I'm injecting following CSS from my custom logo.
It removes the sidebar on pages containing a Document field, but since it uses `has()` it does not work in Firefox.
```css
div[class$="-ColumnLayout"] 
> div[class$="-ColumnLayout"]:has(div[class$="-DocumentEditor"]){
	display: flex;	
}

div[class$="-ColumnLayout"] 
> div[class$="-ColumnLayout"] 
> div:has(div[class$="-DocumentEditor"]) {
	width: 100%;
}

div[class$="-ColumnLayout"]:has(div[class$="-DocumentEditor"]) 
> div[class$="-StickySidebar"] {
	display: none;
}

```

**Comment by johnnycrich** [comment]
Just adding a comment of support for this feature in the future. My users complain about the document field not using the full width of the window, and I find the item ID column to be superfluous.
